css/style.css

```diff
@@ -1,3 +1,51 @@
-.book {
-  margin: 5px 0 5px 0;
+.bg-grey {
+  background-color: #ddd;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+main {
+  width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
```

Comment:
- [ ]   Let's keep our code DRY. `display: flex` and `flex-direction: column` has been repeated more than once. To solve this you can group all classes with the same attribute together and give it one attribute eg
```
main,
.add-btn {
    display:flex
}
```